tests/nnstreamer_filter_tensorflow/runTest.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author HyoungJoo Ahn <hello.ahn@samsung.com>
+## @date Dec 17 2018
+## @brief SSAT Test Cases for NNStreamer
+##
+
```

Comment:
Would it be recommended to add what https://github.com/nnsuite/nnstreamer/pull/992 has done here?
@wooksong

Reply:
@helloahn Please do not use '/'.